packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -249,7 +249,7 @@ export async function readConfigFromDisk() {
             rootPath = vscode.workspace.workspaceFolders[0].uri.fsPath;
             await mProfileInfo.readProfilesFromDisk({ homeDir: getZoweDir(), projectDir: getFullPath(rootPath) });
         } else {
-            await mProfileInfo.readProfilesFromDisk({ homeDir: getZoweDir() });
+            await mProfileInfo.readProfilesFromDisk({ homeDir: getZoweDir(), projectDir: "" });
         }
```

Comment:
should we simplify this if/else 😋 (similar to other places)
`vscode.workspace.workspaceFolders?.[0].uri.fsPath ?? "",`

Reply:
This is in the ZE API ProfilesCache so no vscode specific code in this file.  There will need to be a check for this.cwd being the OS's home directory if possible

Reply:
I'd like to make that simplification but am not sure if I can here because the `getFullPath` method is called inside the if statement. I suppose this raises the question of whether `getFullPath` should be called in the file above (`ZoweExplorerExtender.ts`), because it isn't currently?

Reply:
good catch! 🤔 
Maybe it should be called in both since we've had issues with windows file paths before 👍